src/components/project-card/index.css

```diff
@@ -0,0 +1,41 @@
+.project-card {
+  align-items: center;
+  background-color: #3b7227b2;
+  border-radius: 20px;
+  border: solid 7px #529546;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  height: 100%;
+  justify-content: center;
+  padding: 20px 10px 20px 10px;
+  width: 100%;
+}
+
+.project-image-container {
+  align-items: flex-end;
+  background-position: center;
+  background-size: cover;
+  border-radius: 20px;
+  display: flex;
+  font-family: "Malayalam MN";
+  justify-content: flex-end;
+  margin-bottom: 10px;
+  min-height: 55%;
+  padding: 2px 20px;
+  text-align: right;
+  text-transform: uppercase;
+  width: 75%;
+}
+
+.description {
+  align-items: center;
+  color: white;
+  display: flex;
+  font-family: "Malayalam MN";
+  font-weight: 400;
+  line-height: 15px;
+  size: 10px;
+  text-align: center;
+  width: 80%;
+}
```

Comment:
instead of creating css classes, use the in-built tailwind classes, reference https://tailwindcss.com/ 

you can see how these classes are used via their examples such as https://tailwindcss.com/docs/border-width

it'll be like className="font-3xl"

we'll go more in depth about this at tomorrow's meeting